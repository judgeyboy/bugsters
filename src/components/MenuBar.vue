<template>
  <div class="menu-bar">
    <div class="menu-bar__company">
      <span>
        <router-link to="/">BUGSTERS</router-link>
      </span>
    </div>
    <div class="menu-bar__dashboard">
      <span>DASHBOARD</span>
    </div>
  </div>
</template>

<script>
export default {
  name: 'MenuBar'
}
</script>

<style lang="sass" scoped>
.menu-bar
  height: 50px
  display: flex
  justify-content: space-between
  background-color: $background-grey
  padding: 40px 100px 60px 100px

  &__company
    display: flex
    align-items: center

    span
      font-size: 24px
      font-weight: 100

  &__dashboard
    display: flex
    align-items: center

    span
      font-size: 24px
      font-weight: 100

</style>
