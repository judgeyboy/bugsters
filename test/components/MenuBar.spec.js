import { shallow } from 'vue-test-utils'

import MenuBar from '@/components/MenuBar'

describe('MenuBar component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(MenuBar)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(MenuBar)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
