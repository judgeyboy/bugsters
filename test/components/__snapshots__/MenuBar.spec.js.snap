// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuBar component should match snapshot 1`] = `

<div class="bugster-menubar">
  <div class="bugster-menubar__menu">
  </div>
  <div class="bugster-menubar__link">
  </div>
</div>

`;
